packages/libczmq/01-support-ifaddrs.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/include/czmq_prelude.h b/include/czmq_prelude.h
+index 8a1d7ea7..785ae052 100644
+--- a/include/czmq_prelude.h
++++ b/include/czmq_prelude.h
+@@ -286,9 +286,14 @@
+ #   include <sys/un.h>
+ #   include <sys/uio.h>             //  Let CZMQ build with libzmq/3.x
+ #   include <netinet/in.h>          //  Must come before arpa/inet.h
+-#   if (!defined (__UTYPE_ANDROID)) && (!defined (__UTYPE_IBMAIX)) \
+-    && (!defined (__UTYPE_HPUX))
+-#       include <ifaddrs.h>
++#   if (!defined (__UTYPE_IBMAIX)) && (!defined (__UTYPE_HPUX))
++#       if(defined(__UTYPE_ANDROID))
++#           if(defined(__ANDROID_API__)) && (__ANDROID_API__ >= 24)
```

Comment:
Just keep the original ifdef, but remove Android. No need to add all this, this repo requires API 24.

Reply:
I will do that